### PR TITLE
Fix policy conflict in the CPU manager node e2e test.

### PR DESCRIPTION
**Pull request by balajismaniam**
**What this PR does / why we need it**:
After graduation of the CPU manager feature to Beta, the CPU manager `none` policy is ON by default. But when the CPU manager is set to use `static` policy in the node e2e test, there will always be a conflict with the policy checkpointed in the disk. This PR fixes that by deleting the state file where required. 

Manually tested in an `n1-standard-4` instance with `Ubuntu 16.04` image on GCP, which is the same machine and image type as one of the configs used in the node e2e tests. 

Use the following command to run the test locally:
`make test-e2e-node TEST_ARGS='--feature-gates=DynamicKubeletConfig=true' FOCUS="CPU Manager" SKIP="" PARALLELISM=1`

CC @ConnorDoyle @derekwaynecarr

### CODE REVIEW

File: test/e2e_node/cpu_manager_test.go
Comment (balajismaniam) [inline test/e2e_node/cpu_manager_test.go]: This is not required anymore as the CPU manager feature is ON by default after graduation to Beta.

File: test/e2e_node/cpu_manager_test.go
Comment (ConnorDoyle) [inline test/e2e_node/cpu_manager_test.go]: Could we either make a deleteStateFile helper or move the state path to a constant? Would prefer for that string to live in one place.

**Review by ConnorDoyle — CHANGES_REQUESTED** [review]
LGTM after minor tweak below

File: test/e2e_node/cpu_manager_test.go
Comment (balajismaniam) [reply test/e2e_node/cpu_manager_test.go]: I moved it to a helper func.

**Comment by balajismaniam** [comment]
@ConnorDoyle Updated and tested manually again. PTAL.

**Comment by balajismaniam** [comment]
Unrelated flake. 
/retest

**Review by ScorpioCPH — COMMENTED** [review]
Thanks, I have tested this patch on local, LGTM :)

**Comment by balajismaniam** [comment]
Unrelated UT failure.
/retest

**Comment by balajismaniam** [comment]
@derekwaynecarr @dchen1107 @yujuhong can I get a no-issue approval on this PR? It fixes https://k8s-testgrid.appspot.com/wg-resource-management#kubelet-serial-gce-e2e-cpu-manager.

**Comment by ConnorDoyle** [comment]
/lgtm

@balajismaniam we can either lobby for an `/approve no-issue` or create an issue and link it from the PR description.

**Pull request merged**
